data/json/mapgen_palettes/lodge_palette.json

```
@@ -6,7 +6,7 @@
       "#": "t_rock_wall",
       "+": [ [ "t_door_c", 3 ], "t_door_locked" ],
       "=": "t_door_c",
-      "z": "t_door_metal_locked",
+      "z": "t_door_metal_pickable",
```

```suggestion
      "z": "t_door_metal_interior_locked",
```
I specifically didn't use the door you've selected as your change would have made it unlocked on it's inside sides.  Which are both sides.  Now that someone has added a pickable metal door that doesn't open to both interior sides this can be changed.